cura/MachineActionManager.py

```diff
@@ -38,11 +38,10 @@ def initialize(self):
         # Ensure that all containers that were registered before creation of this registry are also handled.
         # This should not have any effect, but it makes it safer if we ever refactor the order of things.
         for container in container_registry.findDefinitionContainers():
```

Comment:
It's not your change, but could you change this into:
```suggestion
        for container in container_registry.findDefinitionContainersMetadata():
```

That could save a bit of boot time.